landa/organization_management/doctype/member_function/member_function.py

```diff
@@ -102,6 +103,32 @@ def is_inactive(self):
 		return self.end_date and date_diff(today(), self.end_date) > 0
 
 
+	def validate_unique_roles(self):
+		unique_roles = frappe.get_value(
+			"Member Function Category", self.member_function_category, "only_one_per_organization"
+		)
+
+		member_function_category_name = frappe.get_value(
+			"Member Function Category", self.member_function_category, "name"
+		)
+
+		if unique_roles == 1:
+			existing_member_functions = frappe.get_all(
+				"Member Function",
+				filters={
+					"organization": self.organization,
+					"member": self.member,
+					"member_function_category": self.member_function_category,
+					"name": ["!=", self.name],  # Exclude this document from the search
+				},
+			)
+			if existing_member_functions:
+				frappe.throw(
+					_(
+						"The function category can only be assigned once per member in the same organization."
+					).format(member_function_category_name)
+				)
```

Comment:
Ref: https://www.w3schools.com/python/ref_string_format.asp
```suggestion
				frappe.throw(
					_(
						"The {0} Function Category can only be assigned once per member in the same organization."
					).format(frappe.bold(member_function_category_name))
				)
```